### PR TITLE
add cookies

### PR DESCRIPTION
**Pull request by kalefranz**
Python cookies is a requirement for the forthcoming `responses` recipe I'll be submitting, which will be a new test requirement for conda.

### CODE REVIEW

**Comment by kalefranz** [comment]
Any help with the linter error here?

And also, what are you guys doing now with PyPI's terrible URL hashes?

File: recipes/cookies/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "cookies" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/cookies/meta.yaml]: Yep, that's what we are doing. 😄

Comment (kalefranz) [inline recipes/cookies/meta.yaml]: Yeah got it from one of your recipes just a bit ago.

File: recipes/cookies/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - pytest
+  commands:
+    - 'py.test {{ SRC_DIR }}{{ os.sep }}test*'
+
+about:
+  home: https://gitlab.com/sashahart/cookies
+  license: MIT
+  summary: >
+    Friendlier RFC 6265-compliant cookie parser/renderer.  It can be used as
+    a replacement of Python’s Cookie.py (aka http.cookies).
```
Comment (ocefpaf) [inline recipes/cookies/meta.yaml]: The linter might be choking on that unicode `'`. (I thought that was fixed :confused:)

Comment (kalefranz) [inline recipes/cookies/meta.yaml]: Oh I see it now. Was just a simple copy and paste.

**Comment by pelson** [comment]
Failing on Windows:

```
The following NEW packages will be INSTALLED:
    colorama:       0.3.7-py27_0   conda-forge
    cookies:        2.2.1-py27_0   local      
    py:             1.4.31-py27_0  defaults   
    pytest:         2.9.2-py27_0   defaults   
    python:         2.7.11-4       defaults   
    vs2008_runtime: 9.00.30729.1-1 defaults   
C:\Miniconda\conda-bld\test-tmp_dir>py.test C:\Miniconda\conda-bld\work\cookies-2.2.1\test* 
============================= test session starts =============================
platform win32 -- Python 2.7.11, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: C:\Miniconda\conda-bld\work\cookies-2.2.1, inifile: 
====================== 1 pytest-warnings in 0.00 seconds ======================
ERROR: file not found: C:\Miniconda\conda-bld\work\cookies-2.2.1\test*
C:\Miniconda\conda-bld\test-tmp_dir>if errorlevel 1 exit 1 
TESTS FAILED: cookies-2.2.1-py27_0
Command exited with code 1
```

**Comment by jakirkham** [comment]
Toggling to retrigger CIs.

File: recipes/cookies/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/cookies/meta.yaml]: Seems to be fixed in the recent lint.

**Comment by jakirkham** [comment]
Superseded by PR ( https://github.com/conda-forge/staged-recipes/pull/6617 ).

**Pull request closed without merging**